bin/reth/Cargo.toml

```diff
@@ -10,6 +10,10 @@ build = "build.rs"
 [build-dependencies]
 built = { version = "0.6", features = ["git2", "chrono", "semver"] }
 
+# Add jemalloc for extra perf on Linux systems.
+[target.'cfg(all(not(windows), not(target_env = "musl")))'.dependencies]
+jemallocator = "0.5.0"
```

Comment:
looking at lighthouse, they have a dedicated malloc utils crate where jemalloc is a feature and the main binary enables that feature

https://github.com/sigp/lighthouse/blob/693886b94176faa4cb450f024696cb69cda2fe58/lighthouse/Cargo.toml#L27-L28

there are some additional comments here about background threads which are perhaps not relevant?

https://github.com/sigp/lighthouse/blob/stable/common/malloc_utils/Cargo.toml#L14-L16

Reply:
We could do that as a future refactor, looking for a quick win here